default/stringtables/en.txt

```diff
@@ -8887,7 +8887,7 @@ BLD_LIGHTHOUSE
 Interstellar Lighthouse
 
 BLD_LIGHTHOUSE_DESC
-Increases speed of ships departing this system, and decreases the [[encyclopedia STEALTH_TITLE]] of all objects in the same system by 30. This building can be built at an [[encyclopedia OUTPOSTS_TITLE]]. Friendly ships within 50 uus gain a [[encyclopedia STARLANE_SPEED_TITLE]] bonus of 20.
+Increases speed of ships departing this system, and decreases the [[encyclopedia STEALTH_TITLE]] of all objects in the same system by 30. Friendly ships within 50 uus gain a starlane speed bonus of 20. This building can be built at an [[encyclopedia OUTPOSTS_TITLE]] and may facilitate use of a [[buildingtype BLD_PLANET_DRIVE]].
```

Comment:
This removed a Pedia link?  Also, I've never liked that wording, as we're fixing it might as well make it accurate, newly built ships don't get teh bonus on their first turn, so...

"Decreases the [[encyclopedia STEALTH_TITLE]] of all objects in the same system by 30 and increases the [[encyclopedia STARLANE_SPEED_TITLE]] of friendly ships that start the turn within 50uu by 20 for that turn. This building can be built at an [[encyclopedia OUTPOSTS_TITLE]] and may facilitate use of a [[buildingtype BLD_PLANET_DRIVE]]."